### PR TITLE
[16.0][MIG] account_statement_import_online_paypal: Migration to version 16.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT45361

@victoralmau @NL66278 please review!

This PR depends on: #548

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]
Code review OK

**Comment by pedrobaeza** [comment]
CI is red

**Comment by NL66278** [comment]
@stefan-tecnativa You might want to rebase this one now 548 is merged.

**Review by NL66278 — APPROVED** [review]
:+1:  LGTM

File: account_statement_import_online_paypal/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Online Bank Statements: PayPal.com",
-    "version": "15.0.1.1.1",
+    "version": "16.0.1.1.1",
```
Comment (pedrobaeza) [inline account_statement_import_online_paypal/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by pedrobaeza — COMMENTED** [review]
Squash administrative commits https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**